fe/src/components/global/wallet-button.vue

```diff
@@ -83,7 +85,14 @@ const connectWallet = async (): Promise<void> => {
 
 const disconnectWallet = (): void => {
   updateStatus(false, "");
-  toast.success("Wallet disconnected successfully!", { autoClose: 1500 });
+  useCookie("is-connected").value = null;
+
+  toast.success("Wallet disconnected successfully!", {
+    autoClose: 1500,
+  });
+  setTimeout(() => {
+    router.push("/");
+  }, 1500);
```

Comment:
this next line will not be executed:
`localStorage.removeItem("is-connected");`

Reply:
what do you mean by this sir? 
localstorage.removeItem('is-connected') still ran as per checking devtools.